teamwork/apps/projects/migrations/0006_auto_20190125_2151.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-

```

Comment:
Don't merge these into production until this PR gets the updates from the squash PR

Reply:
Done